extcap_ot.py

```diff
@@ -46,29 +45,34 @@ def extcap_dlts(interface):
     print("dlt {number=195}{name=IEEE802_15_4_WITHFCS}{display=IEEE 802.15.4 with FCS}")
     print("dlt {number=283}{name=IEEE802_15_4_TAP}{display=IEEE 802.15.4 TAP}")
 
-def serialopen(interface_port, STDOUT, DirtylogFile):
+def serialopen(interface, __console__, DirtylogFile):
+    """
+    Open serial to indentify OpenThread sniffer
+    :param interface: string, eg: "/dev/ttyUSB0 - Zolertia Firefly platform", "/dev/ttyACM1 - nRF52840 OpenThread Device"
+    """
     sys.stdout = DirtylogFile
-    stream = StreamOpen('u', str(interface_port).split()[0], False)
+    interface = str(interface).split()[0]
+
+    stream = StreamOpen('u', interface, False)
     wpan_api = WpanApi(stream, nodeid=DEFAULT_NODEID)
     value = wpan_api.prop_get_value(SPINEL.PROP_CAPS)
 
     if value is not None:
-        sys.stdout = STDOUT
+        sys.stdout = __console__
         if sys.platform == 'win32':
-            print("interface {value=%s}{display=OpenThread Device %s}" % (str(interface_port).split()[0], str(interface_port).split()[0]))
+            print("interface {value=%s}{display=OpenThread Sniffer %s}" % (interface, interface))
         else:
-            print("interface {value=%s}{display=OpenThread Device}" % str(interface_port).split()[0])
-        sys.stdout = DirtylogFile
+            print("interface {value=%s}{display=OpenThread Sniffer}" % interface)
```

Comment:
when run below commands with one sniffer device and one cli device (serial opened), below errors are output in console. may need redirect stderr

> ➜  pyspinel git:(pr71_2) ✗ ./extcap_ot.py --extcap-interfaces
> extcap {version=0.0.0}{display=OT Sniffer}{help=https://github.com/openthread/pyspinel}
> ERROR:root:Couldn't open /dev/cu.usbmodemC3331B42220F2
> Traceback (most recent call last):
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/stream.py", line 58, in __init__
>     self.serial = serial.Serial(dev, baudrate)
>   File "/Users/rongli/Library/Python/2.7/lib/python/site-packages/serial/serialutil.py", line 240, in __init__
>     self.open()
>   File "/Users/rongli/Library/Python/2.7/lib/python/site-packages/serial/serialposix.py", line 268, in open
>     raise SerialException(msg.errno, "could not open port {}: {}".format(self._port, msg))
> SerialException: [Errno 16] could not open port /dev/cu.usbmodemC3331B42220F2: [Errno 16] Resource busy: '/dev/cu.usbmodemC3331B42220F2'
> Exception in thread Thread-7:
> Traceback (most recent call last):
>   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
>     self.run()
>   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 763, in run
>     self.__target(*self.__args, **self.__kwargs)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/codec.py", line 918, in stream_rx
>     self.rx_pkt = self.hdlc.collect()
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/hdlc.py", line 80, in collect
>     byte = self.stream.read()
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/stream.py", line 69, in read
>     pkt = self.serial.read(size)
> AttributeError: 'StreamSerial' object has no attribute 'serial'
> 
> Exception in thread Thread-4:
> Traceback (most recent call last):
>   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
>     self.run()
>   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 763, in run
>     self.__target(*self.__args, **self.__kwargs)
>   File "./extcap_ot.py", line 58, in serialopen
>     value = wpan_api.prop_get_value(SPINEL.PROP_CAPS)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/codec.py", line 1062, in prop_get_value
>     None, None, tid)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/codec.py", line 1047, in __prop_change_value
>     self.transact(cmd, pay, tid)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/codec.py", line 871, in transact
>     self.stream_tx(pkt)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/codec.py", line 912, in stream_tx
>     self.stream.write(pkt)
>   File "/Users/rongli/Documents/stash/pyspinel/spinel/stream.py", line 64, in write
>     self.serial.write(data)
> AttributeError: 'StreamSerial' object has no attribute 'serial'
> 
> interface {value=/dev/cu.usbmodemF37D7A028A7F2}{display=OpenThread Sniffer}
> ➜  pyspinel git:(pr71_2) ✗ ls
> AUTHORS            MANIFEST.in        STYLE_GUIDE.md     requirements.txt   spinel-cli.py
> CODE_OF_CONDUCT.md Makefile.am        dirtylog           setup.py           test_spinel.py
> CONTRIBUTING.md    README.md          extcap_ot.bat      sniffer.py
> LICENSE            SNIFFER.md         extcap_ot.py       spinel
> ➜  pyspinel git:(pr71_2) ✗ ls -lt dirtylog
> -rw-r--r--  1 rongli  primarygroup  0 22 Sep 23:40 dirtylog